modules/distribution/src/main/assembly/bin.xml

```diff
@@ -167,5 +167,12 @@
         <include>*</include>
       </includes>
     </fileSet>
+    <fileSet>
+      <directory>${project.basedir}/../../contrib/grafana</directory>
+      <outputDirectory>conf/grafana</outputDirectory>
```

Comment:
Maybe instead of adding info to `docs/metrics.md` about grafana, there could be a `contrib/grafana/readme.md`.   However maybe this does not work well for the source on GH and the tarball layout.